plugins/builds/create.js

```diff
@@ -112,7 +112,8 @@ module.exports = () => ({
                                         type,
                                         username,
                                         scmContext,
-                                        sha
+                                        sha,
+                                        skipMessage: 'skip build creation'
```

Comment:
why can't we just remove lines 119 - 130. ie don't create a build here let eventFactory take care if it. ?

Reply:
Because following logic needs a build and eventFactory return an event, thus we would need to do a build look up. There might be a cleaner way but I am looking to fix this ASAP to unblock other people